app/api/rest_api.py

```diff
@@ -77,52 +50,22 @@ def __init__(self, services):
     async def logout(self, request):
         await self.auth_svc.logout_user(request)
 
-    """ SPLIT SECTIONS """
-
     async def landing(self, request):
-        return await self.get_endpoint_by_access(request, 'landing')
-
-    async def section_agent(self, request):
-        return await self.get_endpoint_by_access(request, 'section_agent')
-
-    async def section_profiles(self, request):
-        return await self.get_endpoint_by_access(request, 'section_profiles')
-
-    async def section_operations(self, request):
-        return await self.get_endpoint_by_access(request, 'section_operations')
-
-    async def section_sources(self, request):
-        return await self.get_endpoint_by_access(request, 'section_sources')
-
-    async def get_abilities(self, request):
-        return await self.get_endpoint_by_access(request, 'get_abilities')
+        access = [p for p in await self.auth_svc.get_permissions(request)]
```

Comment:
you are already returning an array cant this just be `access = await self.auth_svc.get_permissions(request)`
the only reason it was a comprehension before was for the conditional clause at the end.

Reply:
yep!